app/build.gradle.kts

```diff
@@ -0,0 +1,189 @@
+plugins {
+    alias(libs.plugins.androidApplication)
+    alias(libs.plugins.jetbrainsKotlinAndroid)
+    alias(libs.plugins.ktfmt)
+    alias(libs.plugins.sonar)
+    id("jacoco")
+}
+
+android {
+    namespace = "com.android.sample"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "com.android.sample"
+        minSdk = 28
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
```

Comment:
Maybe we should already define those before merging to main?

Reply:
I will let them as they are right now until we know the name of the app. We can still change this later